bin/good-first-issue.js

```diff
@@ -34,15 +34,22 @@ cli
       console.log("If you'd like to add a new project to good-first-issue,")
       console.log(
         "Please see the module's Github repository: " +
-          chalk.cyan(
-            'https://github.com/bnb/good-first-issue#adding-new-projects'
-          )
+        chalk.cyan(
+          'https://github.com/bnb/good-first-issue#adding-new-projects'
+        )
       )
       console.log('')
       process.exit(0)
     }
 
-    const issues = await goodFirstIssue(input)
+    var issues
+
+    try {
+      issues = await goodFirstIssue(input)
+    } catch (e) {
+      console.log('Oops! We encountered an issue while searching. Please check your network connection.')
+      process.exit(1)
+    }
 
     if (issues.length === 0) {
```

Comment:
If at this point we were to intercept an empty string literal from the response header rather than the results of a list, and this string literal was to be an error response in a short-circuit and no error was able to be parsed, edge of timeouts, parsing errors etc. This would still return as true, hence returning an empty string exposed.
It would be suitable at some point here validate the actual type integrity of the expected data formats being used, as side effects of this may lead to false positives.